src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/activity/WallPostActivity.java

```diff
@@ -0,0 +1,29 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.activity;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+
+public class WallPostActivity extends Activity {
+
+  private WebElement diffLink;
+  private WebElement activityDescriptionTextBox;
+
+  public WallPostActivity(WebDriver driver, WebElement parentElement) {
+    super(driver, parentElement);
+
+    By diffButtonSelector = By.cssSelector("a.activityfeed-diff");
+    diffLink = parentElement.findElement(diffButtonSelector);
+    By articleDescriptionTextBoxSelector = By.cssSelector("tbody td:nth-child(2) p:nth-child(2)");
```

Comment:
Magic number. Maybe we can add it as constant with descriptive name above that line ?